### PR TITLE
chore(docs): Update getting started for webpack CLI

### PR DESCRIPTION
**Pull request by davideast**
Updated to the latest and greatest with that webpack goodness.

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -1,113 +1,39 @@
...
-import { MdCardModule } from '@angular2-material/card';
-import { MdButtonModule } from '@angular2-material/button';
-```
-
-Import the components in your application module:
-
-**my-app-module.ts**
+**src/app/app.module.ts**
 ```ts
+import { MdButtonModule } from '@angular2-material/button/button';
```
Comment (m0t0r) [inline GETTING_STARTED.md]: @davideast `@angular2-material/button/button` was moved to `'@angular2-material/button'` in the latest `alpha.8`, the same for all components.

**Comment by jelbourn** [comment]
@davideast we should remove the references to the puppy-love app while we're at it, since it's _really_ out of date now.

**Comment by davideast** [comment]
@jelbourn Done.

**Comment by jelbourn** [comment]
LGTM

**Pull request merged**